bears/__init__.py

```diff
@@ -20,3 +20,25 @@ def assert_supported_version():  # pragma: no cover
     if sys.version_info < (3, 4):
         print('coala supports only python 3.4 or later.')
         exit(4)
+
+
+def check_coala_version():
+    """
+    Check the installed coala and coala-bears version when the coalib loads
+    coala-bears module.
+    """
+    try:
+        import coalib
+        if coalib.__version__ != __version__:
```

Comment:
this is too simple.

coala-bears version might be greater than coala version.  Isnt that OK?
You need to do a proper version comparison.  There are classes that make this very easy.